grafana_dashboard_manager/dashboard.py

```diff
@@ -89,7 +89,7 @@ def update_single_dashlist_folder_id(panels_definition: Dict) -> Dict:
         logger.info(f"{folder_name} folder has ID {folder_id}")
 
         # Ensure that the folder id in the dashboard definition matches
-        if panels_definition["options"]["folderId"] == folder_id:
+        if "folderId" in panels_definition["options"].keys() and panels_definition["options"]["folderId"] == folder_id:
```

Comment:
```suggestion
        if panels_definition["options"].get("folderId"):
```

would probably be more Pythonic.